content/release-10-15-0/devices-10-15-0-bundle/10-15-0-0.md

```diff
@@ -7,19 +7,27 @@ layout: bundle
 <!--10.14.1.0 - 10.14.105.0-->
 
 ### Improvements
-
-<div><table ><colgroup>
-<col style="width: 15%;"><col style="width: 70%;"><col style="width: 15%;"></colgroup>
-<thead><tr>
-<th>
-Component</th>
-<th>
-Description</th>
-<th>
-Issue</th>
-</tr>
+
+<div><table ><colgroup>
+<col style="width: 15%;"><col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Component</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
 </thead><tbody>
 
+
+<tr>
+<td>Actility</td>
```

Comment:
If you add entries manually here, please make sure that they are sorted correctly (alphabetically). Moved the entry for Actility accordingly. Moreover I moved both entries to the Improvements section as they are not a bug fix.

Reply:
Sure, Thank you.